src/dataview64.js

```diff
@@ -46,6 +52,10 @@ export default class DataView64 {
     return this._dataView.getInt16(offset, littleEndian);
   }
 
+  getUint24(offset) {
```

Comment:
I'm not sure if we actually need this function. Also, the big endian case has to be handled as-well.